src/components/Card/Subtitle.tsx

```diff
@@ -0,0 +1,11 @@
+import { Sprinkles, Text } from "@saleor/macaw-ui-next";
+import React, { PropsWithChildren } from "react";
+
+type CardSubtitleProps = Sprinkles;
+
+export const CardSubtitle: React.FC<PropsWithChildren<CardSubtitleProps>> = ({ children }) =>
+  children ? (
+    <Text size={4} fontWeight="light" display="block">
```

Comment:
question: what about passing rest of Text props, for example to move text if needed?